Makefile

```diff
@@ -19,16 +19,20 @@ build-cli: clean-tags
 	./build-php.sh cli 7.2 3.7
 	./build-php.sh cli 7.2 3.8
 	./build-php.sh cli 7.2 3.9
+	./build-php.sh cli 7.2 3.10
```

Comment:
There is no `3.10` tag available upstream: https://hub.docker.com/_/php

Reply:
I suppose we'll just have to wait for it then.

Reply:
It is now, this is passing and merged :tada: